client/views/room/hooks/useUserInfoActions.js

```diff
@@ -310,7 +310,7 @@ export const useUserInfoActions = (user = {}, rid) => {
 			text={t('The_user_will_be_removed_from_s', roomName)}
 			close={closeModal}
 			confirmText={t('Yes_remove_user')}
-			confirm={() => { removeUserAction({ roomId: rid, userId: uid }); closeModal(); }}
+			confirm={() => { removeUserAction({ roomId: rid, username: user.username }); closeModal(); }}
```

Comment:
Although this works for the user removal, it's not the ideal fix, since `uid` is used in other places that should be broken too. Try to see where this user object is coming from and why the `_id` property is not coming along with it.

This issue looks pretty bad, so I'll give you another couple of hours to figure it out, otherwise I'll fix it.

Reply:
Working on it!